src/Bicep.Cli.IntegrationTests/GenerateParamsCommandTests.cs

```diff
@@ -0,0 +1,347 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Skip putting parameters into the params file that have a default value that is an expression